config/_default/config.yaml

```diff
@@ -100,3 +100,6 @@ security:
     getenv:
       # allow os.Getenv to read gitlab CI_ prefixed os environment vars
       - ^(HUGO_|CI_)
+  exec:
+    osEnv:
```

Comment:
do we need these security `osEnv` definitions in both default and development config?   might be worth adding a few other variables as suggested [here](https://github.com/gohugoio/hugo/issues/9333#issuecomment-1022435616)?

Reply:
I had no problems with the build Nick added this to resolve m1 mac issues for him. I've set this to just be in default with the full version you are suggesting.